specs/consensus.md

```diff
@@ -307,14 +308,13 @@ Apply the following to the state:
 
 ```py
 state.accounts[sender].nonce += 1
-state.accounts[sender].balance -= totalCost(tx.amount, bytesPaid)
+state.accounts[sender].balance -= totalCost(0, bytesPaid)
 state.accounts[sender].status = AccountStatus.ValidatorQueued
 
 validator = new Validator
-validator.stakedBalance = tx.amount
 validator.commissionRate = tx.commissionRate
 validator.delegatedCount = 0
-validator.votingPower = tx.amount
+validator.votingPower = 0
```

Comment:
What is the benefit here?

Reply:
This is part of the refactor of removing staked balance for validators:

> Remove `stakedBalance` for validators. Validators now have no balance and need to delegate to themselves. This can be abstracted away at the UI layer.

(Note that this does introduce a divide-by-zero case when computing entries, but I'll take care of that in a future PR.)